File: app/src/main/java/com/websarva/wings/android/kaigonote/SyokuzennyakuasaActivity.java
package com.websarva.wings.android.kaigonote;

import android.os.Bundle;
import android.view.View;

import androidx.appcompat.app.AppCompatActivity;

import com.websarva.wings.android.kaigonote.databinding.SyokuzennyakuasaBinding;

public class SyokuzennyakuasaActivity extends AppCompatActivity {
    private SyokuzennyakuasaBinding binding;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        binding = SyokuzennyakuasaBinding.inflate(getLayoutInflater());
        View view = binding.getRoot();
        setContentView(view);
    }
}

File: app/src/main/java/com/websarva/wings/android/kaigonote/SyokuzennyakuasaActivity.kt
package com.websarva.wings.android.kaigonote

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.view.View
import com.websarva.wings.android.kaigonote.databinding.SyokuzennyakuasaBinding

class SyokuzennyakuasaActivity : AppCompatActivity() {
    private var binding: SyokuzennyakuasaBinding? = null
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        binding = SyokuzennyakuasaBinding.inflate(layoutInflater)
        val view: View = binding!!.root
        setContentView(view)
    }
}